manifests/l/LWorks/SuperEggHunt/1.53/LWorks.SuperEggHunt.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LWorks.SuperEggHunt
+PackageVersion: 1.53

```

The version shown in the DisplayName shows 1.6
```suggestion
PackageVersion: 1.6
```